homeassistant/util/unit_conversion.py

```diff
@@ -284,6 +299,28 @@ class EnergyConverter(BaseUnitConverter):
     VALID_UNITS = set(UnitOfEnergy)
 
 
+class EnergyDistanceConverter(BaseUnitConverter):
+    """Utility to convert vehicle energy consumption values."""
+
+    UNIT_CLASS = "energy_distance"
+    _UNIT_CONVERSION: dict[str | None, float] = {
+        UnitOfEnergyDistance.KILO_WATT_HOUR_PER_100_KM: 1,
+        UnitOfEnergyDistance.MILES_PER_KILO_WATT_HOUR: 100 * _KM_TO_M / _MILE_TO_M,
+        UnitOfEnergyDistance.KM_PER_KILO_WATT_HOUR: 100,
+    }
+    _UNIT_INVERSES: set[tuple[str, str]] = {
```

Comment:
Could this not be a simple `set[str]`?

If `(unit A in set) != (unit B in set)` then invert

Reply:
Maybe, but I'm not sure that would be easier to read so I'd choose the more apparent solution.